### ISSUE TITLE
Noise maps

### ISSUE THREAD

**Issue by rosella1234** [samuelstjean/nlsam#133]
Hi
is it possible to obtain noise maps resulting from nlsam denoising?
best,
Rosella

**Comment by samuelstjean** [comment]
You can save the estimated noise standard deviation by passing the option **--save_sigma my_noise_map.nii.gz**

**Comment by rosella1234** [comment]
Thank you. I have added "--save_sigma" command to my nlsam denoising and I obtain this kind of map: 
![image](https://github.com/user-attachments/assets/ba952f85-0e10-44df-9c8d-34a06e2a4b20)
Is it correctly computed as expected?

thanks in advance,

Rosella

**Comment by samuelstjean** [comment]
Yes, it's a 2D map so each slice along the dimension you ran it (usually axial by default) will have the same value when ran over data.
